build.html

```diff
@@ -37,6 +37,7 @@
       };
     </script>
     <div data-script-insert="{{ id }}" class="hidden"></div>
+    <script type="text/javascript" src="{{{ asset "js/query-parser.js" }}}"></script>
```

Comment:
Doesn't this mean the file will be loaded multiple times if there are multiple instances on the page?

Reply:
Yep :) I've commented too that the file should be listed in the widget.json

Reply:
Yes, as this is true for all of the LFD code anyway. This is just a way of abstracting the parser out and not having to repeat in every single layout code.